assets/manifest.json

```diff
@@ -25,22 +25,22 @@
   "content_scripts": [
     {
       "matches": [
-        "https://github.com/*",
```

Comment:
I remove the github domain in order to test the functionality. I'll revert this as soon as the PR is ready to go.